app/src/main/java/org/pesmypetcare/mypetcare/features/pets/PetRepeatException.java

```diff
@@ -0,0 +1,7 @@
+package org.pesmypetcare.mypetcare.features.pets;
+
+public class PetRepeatException extends Exception {
+    public PetRepeatException() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5219990)